test/functionalTest/cases/1188_empty_attribute_value/empty_attribute_value.test

```diff
@@ -0,0 +1,194 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Empty Attribute Value for API v2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v2/entities to create E1/A1=0
+# 02. GET /v2/entities/E1 to see A1==0
+# 03. PUT /v2/entities/E1/attrs/A1/value (="")
+# 04. GET /v2/entities/E1 to see A1==""
+# 05. PUT /v2/entities/E1/attrs/A1/value (=5)
+# 06. GET /v2/entities/E1 to see A1==5
+#
+
+
+echo "01. POST /v2/entities to create E1/A1=0"
```

Comment:
Ah, this function I got from the issue (1188), didn't really look at the echoes ... :-)